.vscode/settings.json

```diff
@@ -2,5 +2,9 @@
     "lit-plugin.globalAttributes": ["disabled"],
     "files.associations": {
         "*.svg": "html"
-    }
+    },
+    "editor.formatOnSave": true,
+    "csscomb.formatOnSave": true,
```

Comment:
I really don't think these kinds of settings should be versioned (aka forced), because we already have linting (prettier and tsc) happening in CI. It's more of a personal preference kind of thing, though, as any malfunction of the "auto-formatters" is _generally_ easy to revert.

Reply:
Yes, you're right. I'll remove it.